pkg/domain/domain.go

```diff
@@ -1861,6 +1861,22 @@ func (do *Domain) GetPDHTTPClient() pdhttp.Client {
 	return nil
 }
 
+func decodePrivilegeEvent(resp clientv3.WatchResponse) []string {
+	var userList []string
+	if len(resp.Events) > 0 {
```

Comment:
this `if` check is not necessary, because `for` is no-op when `Events` has zero length